client/src/components/content/BpmnModeler/BpmnModeler.jsx

```diff
@@ -75,9 +74,13 @@ const BpmnModeler = () => {
           <div className={styles['bpmn-modeler-container']} id='bpmnview' />
         </div>
       </Content>
-      <ModelerSidebar modeler={modeler} getRobotFile={downloadRobotFile} />
+      <ModelerSidebar robotId={robotId} modeler={modeler} getRobotFile={downloadRobotFile} />
     </Layout>
   );
 };
 
 export default BpmnModeler;
+
+BpmnModeler.propTypes = {
+  robotId: PropTypes.string.isRequired
+};
```

Comment:
I am not quite sure but I think export should always be in the last line.

Reply:
Good point. ESLINT says nothing about this. Nevertheless, I have changed all occurrences in the code and also added it to our coding standards.